test/library/helpers.go

```
@@ -0,0 +1,323 @@
+			return false, nil
+		}
+		return true, nil
+	})
+	AssertWaitPollNoErr(err, fmt.Sprintf("metric output should contain %q", metricString))
+}
+
+func WaitForAvailableRsRunning(oc *OC, rsKind, rsName, namespace, expected string) bool {
+	err := wait.Poll(20*time.Second, 180*time.Second, func() (bool, error) {
+		output, err := oc.AsAdmin().WithoutNamespace().Run("get").Args(rsKind, rsName, "-n", namespace, "-o=jsonpath={.status.availableReplicas}").Output()
+		if err != nil {
+			return false, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 148) but it returns nil

(nilerr)